packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-builder-ui-workspace.jsx

```diff
@@ -7,11 +7,25 @@ import AddStage from '../add-stage';
 import ModifySourceBanner from '../modify-source-banner';
 import { moveStage } from '../../modules/pipeline-builder/stage-editor';
 import { sortableContainer, sortableElement } from 'react-sortable-hoc';
+import { css } from '@mongodb-js/compass-components';
 
 import styles from './pipeline-builder-ui-workspace.module.less';
 
+const stageContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  '&.dragging .add-stage-button': {
+    display: 'none'
```

Comment:
hide add stage button when dragging the stage.

Reply:
When you start dragging a stage there's a jump and I wonder if it is caused by this. Maybe we need a better way to hide it like visibility: 'hidden' or if for some reason that doesn't work then opacity: 0 and pointer-events: 'none'?

Something that will cause it to still take up space.

Reply:
not sure what part jumped, when dropping the stage, did stage jump or the other content of the page?
but, filling up the whole space is a better idea. i added that